webapp/src/components/sidebar/registrationLink.tsx

```diff
@@ -44,7 +44,7 @@ const RegistrationLink = (props: Props) => {
         }
     }
 
-    const registrationUrl = Utils.buildURL('/register?t=' + signupToken, true)
```

Comment:
This looks really wrong, we should fix this behavior in the buildURL function or create another function to build the absolute url. Clearly, having an "absolute" parameter that doesn't work in certain context is totally missleading.

Reply:
Agreed, I created #2901 as a code debt ticket to track that specifically

Reply:
but I don't think it should be a different ticket, I think we should fix it properly right away, instead of fixing it wrongly here, and fix the original function, and probably miss to fix it here.